poppy/poppy_core.py

```diff
@@ -2632,7 +2634,7 @@ def __init__(self, name="unnamed optic", transmission=None, opd=None, opdunits=N
         self.amplitude_file = None
         self.amplitude_header = None
         self.opd_header = None
-
+        self._opd_is_phase_map = False
```

Comment:
would `._opd_in_radians` be clearer? That more directly connects it to the relevant input parameter.

Reply:
Yeah, I suppose so. If we wanted to add OPD in waves later, we could rename it since it's an internal attribute 🙃